### PR TITLE
Feature/user api

### PR DESCRIPTION
**Pull request by liannivin**

### CODE REVIEW

File: spec/controllers/api/hotels_controller_spec.rb [hunk 1]
```diff
@@ -60,7 +60,7 @@ def authorization_header
     it 'returns http status created' do
       post :create, params: {
         name: 'Oro Verde',
-        email: 'drupvon+hotel1@gmail.com',
+        email: 'drupvon+hotel2@gmail.com',
```
Comment (condef5) [inline spec/controllers/api/hotels_controller_spec.rb]: Why did you change field email?

Comment (liannivin) [reply spec/controllers/api/hotels_controller_spec.rb]: It was because of update test for users.

Comment (liannivin) [reply spec/controllers/api/hotels_controller_spec.rb]: If I hadn't change this attributes, test would make errors.
